ApiSurface/Test/TestVersionFile.fs

```diff
@@ -159,7 +159,7 @@ module TestVersionFile =
             |> fun s -> fs.File.WriteAllText (location.FullName, s)
 
             let baseline = ApiSurface [ "something" ; "something else" ]
-            ApiSurface.updateVersionJson baseline typeof<ApiSurface>.Assembly location
+            ApiSurface.updateVersionJson baseline typeof<ApiSurface>.Assembly location _.Open
```

Comment:
I have a bit of a problem with argument ordering here. The `_.Open` function is clearly a dependency being injected, so it should go at the start. But left-to-right typechecking means it has to go after `location` if we want to avoid a type annotation.